vyxal/dictionary.py

```diff
@@ -23167,7 +23177,44 @@
 x^2
 x^3
 x+1
-x-1""".split(
+x-1
+<p>
+</p>
+<s>
+</s>
+<strong>
+</strong>
+<em>
+</em>
+<a href="
+</a>
+<img src="
+</img>
+</br>
+<script>
+</script>
+<style>
+</style>
+<h1>
+</h1>
+<h2>
+</h2>
+<div>
+</div>
+<body>
+<svg>
+</body>
+</svg>
```

Comment:
All of the tags have the closing tag immediately after, except for `<body>` and `<svg>`

Reply:
...

That's not an issue at all. All it does it change the index in the short dictionary.